screens/FavPokemon.tsx

```diff
@@ -50,8 +49,30 @@ export default function FavPokemon({ navigation }: Props) {
           <PropRow left='Height:' right={favPokemon.height} />
           <PropRow left='Base experience:' right={favPokemon.base_experience} />
         </View>
-        <Button title='More Info' onPress={buttonCallback} />
-      </View>
+        <View style={{ marginBottom: 70, marginTop: 50 }}>
```

Comment:
you could extract it to stylesheet. It's always better to use stylesheet as then you pass only number, e.g. `5`, which is a reference to stylesheet object. If you pass object like this then you will always pass the whole object to the native side (Android/iOS code).

Of course if you e.g. want to pass `color` then there's no need to invent a lot of code to use `stylesheet` instead of `{ color: myColor }`. But in such simple cases where you can extract style to stylesheet it's always better to extract it